src/astt_gui/app.py

```diff
@@ -147,6 +152,8 @@ def start_astt_gui():
                     background_thread, cm.antenna_node
                 )
 
+    az_speed = request.form.get("az_speed")
+    el_speed = request.form.get("el_speed")
```

Comment:
Following the code, the AZ/EL speed is obtained during initialization. Is it not supposed to happen during tracking or pointing?

Reply:
Interesting observation, I want to ask Phelo does then the speed change when we start pointing or tracking??

Reply:
Resolved please check from here https://github.com/AbednigoLethole/asst-cam-software/pull/39/files#r1718101198